Test/Case/Model/ShimModelTest.php

```diff
@@ -1,5 +1,6 @@
 <?php
 Configure::write('debug', 2);
+App::import('Shim.Lib', 'Shim');
```

Comment:
why import?

Reply:
`App::uses('Shim', 'Shim.Lib');` didn't work for me - class not found exception.

Reply:
I dont like those import ones though.
Then we better keep things inline and use a few lines of more redundant code IMO.

Reply:
Why `import()` is not good? Redundant code makes it more difficult to maintain and already now there is inconsistency in how the shims are handled - some of them support 'exception' value, some - not. Using `check()` for all shims would make it consistent.

Reply:
import is only for vendor files, for nothing else.
Since it seems to not work properly here, maybe due to constants, maybe drop the constant part then?

Reply:
It seems the problem is only on my localhost.